examples/openthread_posix_example/udp.c

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Looks like this file and `main.c` are just copy pasted  from `posix_example`. would it make sense to just modify the initial example ? I mean simply update the README with a section related to openthread stack and add some Makefile variable to switch to openthread instead of gnrc ?